htdocs/bom/class/bom.class.php

```diff
@@ -792,7 +792,7 @@ public function updateLine($rowid, $qty, $qty_frozen = 0, $disable_stock_change
 					$line->array_options[$key] = $array_options[$key];
 				}
 			}
-			if ($fk_default_workstation > 0 && $line->fk_default_workstation != $fk_default_workstation) {
+			if ($line->fk_default_workstation != $fk_default_workstation) {
```

Comment:
Why removing the test
`$fk_default_workstation > 0 &&`

Reply:
I deleted this because it prevented emptying the field when you did not want to apply the default workstation